pkg/controller/installation/core_controller.go

```diff
@@ -828,47 +868,23 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 			r.SetDegraded("Unable to update OpenShift Network config: failed to read OpenShift network configuration", err, reqLogger)
 			return reconcile.Result{}, err
 		}
+
 		// Get resource before updating to use in the Patch call.
 		patchFrom := client.MergeFrom(openshiftConfig.DeepCopy())
-		// If configured to run in openshift, update the config status with the current state.
+
+		// Update the config status with the current state.
 		reqLogger.WithValues("openshiftConfig", openshiftConfig).V(1).Info("Updating OpenShift cluster network status")
 		openshiftConfig.Status.ClusterNetwork = openshiftConfig.Spec.ClusterNetwork
 		openshiftConfig.Status.ServiceNetwork = openshiftConfig.Spec.ServiceNetwork
 		openshiftConfig.Status.NetworkType = "Calico"
-		if instance.Spec.CalicoNetwork != nil && instance.Spec.CalicoNetwork.MTU != nil {
-			// If specified in the spec, then use the value provided by the user.
-			// This is what the rendering code will have populated into the created resources.
-			openshiftConfig.Status.ClusterNetworkMTU = int(*instance.Spec.CalicoNetwork.MTU)
-		} else if instance.Spec.CalicoNetwork != nil {
-			// If not specified, then use the value for Calico VXLAN networking. This is the smallest
-			// value, so might not perform the best but will work everywhere.
-			openshiftConfig.Status.ClusterNetworkMTU = 1410
-		}
+		openshiftConfig.Status.ClusterNetworkMTU = statusMTU
```

Comment:
0 is not really a valid MTU, should we not set the status in that case?
I guess it would be zero by default.... should the field be switched to a pointer.

Reply:
After chatting about this, sounds like we're good. It'll be hidden because of omitempty on the field.